processors/sshd/sshdprocessor.go

```diff
@@ -124,6 +124,9 @@ func ProcessEntry(config *SshdProcessorer) error {
 	case strings.HasPrefix(config.logEntry, "User "):
 		entryFunc = userTypeLogAuditFn(config)
 		config.metrics.IncLogins(metrics.UnknownLogin, metrics.Failure)
+	case strings.HasPrefix(config.logEntry, "Accepted password"):
+		entryFunc = processAcceptedPasswordEntry
+		config.metrics.IncLogins(metrics.UnknownLogin, metrics.Failure)
```

Comment:
I think this line needs to be removed :)

(`config.metrics.IncLogins(metrics.UnknownLogin, metrics.Failure)`)

Reply:
Also - what do you think about moving this `case` directly below the `case strings.HasPrefix(config.logEntry, "Accepted publickey"):`?

That way, the two success cases are right next to each other.